src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball chosenOne = new Ball();
+        ColorSupplier randomColor = new ColorSupplier();
+        int numbersLimit = 101;
+        chosenOne.setColor(randomColor.getRandomColor());
+        Random random = new Random();
+        int randomNumber = random.nextInt(numbersLimit);
+        chosenOne.setNumber(randomNumber);
+        return chosenOne;
+    }
+}

```

Comment:
Please take a look at solution provided below, it's much more readable and clean:

```java
public class Lottery {
    private static final int MAXIMUM_NUMBER = 100;
    private ColorSupplier colorSupplier = new ColorSupplier();
    private Random random = new Random();

    public Ball getRandomBall() {
        return new Ball(colorSupplier.getRandomColor(), random.nextInt(MAXIMUM_NUMBER) + 1);
    }
}
```

Please implement something similar to this

Reply:
indeed :) many thx